### PR TITLE
ref #116: Add command for counting pending updates

### PR DESCRIPTION
**Pull request by UlrichKu**
| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | no
| New feature?  | yes| BC breaks?    | no
| Deprecations? | no
| Fixed issues  | chameleon-system/chameleon-system#116
| License       | MIT

### CODE REVIEW

File: src/CoreBundle/Command/UpdateCountCommand.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+namespace ChameleonSystem\CoreBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Console command for counting updates that have not been executed yet.
+ */
+class UpdateCountCommand extends Command
```
Comment (umulmrum) [inline src/CoreBundle/Command/UpdateCountCommand.php]: Could we rename the command to CountUpdateCommand to be in line with the existing commands (although your scheme would be better if we had used it from the beginning; in the future I'd like to move these commands to the DatabaseMigrationBundle where the current scheme would be OK)?

File: src/CoreBundle/Command/UpdateCountCommand.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        parent::__construct('chameleon_system:update:count');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDefinition([])
```
Comment (umulmrum) [inline src/CoreBundle/Command/UpdateCountCommand.php]: I think this call can be removed.

File: src/CoreBundle/Command/UpdateCountCommand.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            ->setDescription('Prints the count of all pending updates')
```
Comment (umulmrum) [inline src/CoreBundle/Command/UpdateCountCommand.php]: Please begin the description with "The <info>%command.name%</info> command " and end with a dot.

File: src/CoreBundle/Command/UpdateCountCommand.php (hunk 1 above)
Comment (UlrichKu) [reply src/CoreBundle/Command/UpdateCountCommand.php]: ok

File: src/CoreBundle/Command/UpdateCountCommand.php (hunk 3 above)
Comment (UlrichKu) [reply src/CoreBundle/Command/UpdateCountCommand.php]: html help added

File: src/CoreBundle/Command/UpdateCountCommand.php (hunk 2 above)
Comment (UlrichKu) [reply src/CoreBundle/Command/UpdateCountCommand.php]: ok

**Comment by umulmrum** [comment]
I forgot another thing, sorry. Could you add a service definition for the command? Although it works as is, we always define commands as services (which will also be required in future Symfony versions).

**Comment by UlrichKu** [comment]
Service definition added

**Comment by umulmrum** [comment]
Thanks @UlrichKu

**Pull request merged**